sdml-parse/src/load.rs

```diff
@@ -51,6 +52,9 @@ pub const SDML_FILE_EXTENSION_LONG: &str = "sdml";
 /// The name used for resolver catalog files.
 pub const SDML_CATALOG_FILE_NAME: &str = "sdml-catalog.json";
 
+/// The environment variable used to override resolver catalog file location.
+pub const SDML_CATALOG_FILE_VARIABLE: &str = "SDML_CATALOG";
```

Comment:
I would make this `SDML_CATALOG_FILE`, seems easier to understand, and you named the constant `SDML_CATALOG_FILE_VARIABLE` after all.

Reply:
Makes sense. Updated.